python/aitubertalk/dialogue/dialogue_module.py

```
@@ -0,0 +1,379 @@
+"""
+Dialogue control module for AITuberTalk SDK
+"""
+
+import asyncio
+from datetime import datetime, timedelta
+from typing import Callable, Optional, TYPE_CHECKING
+from uuid import uuid4
+
+from ..core.exceptions import AITuberTalkError, ErrorCode
+from .types import FloorToken, SpeakOptions, FloorState, QueuedParticipant, Message
```

Ruff (0.11.9)

`.types.QueuedParticipant` imported but unused

Remove unused import: `.types.QueuedParticipant`

(F401)